manifests/dev/vsphere-7.0u1/guestcluster/1.15/pvcsi.yaml

```diff
@@ -163,6 +168,10 @@ spec:
           image: vmware.io/syncer:<image_tag>
           args:
             - "--leader-election"
+            - "--supervisor-fss-name=csi-feature-states"
+            - "--supervisor-fss-namespace=vmware-system-csi"
```

Comment:
In the same file in Line: 86, Namespace is configurable `namespace: {{ .PVCSINamespace }}`
However, here it is hard-coded. 
Isn't this conflicting ?

Reply:
Good catch, will change it

Reply:
Is it possible to use template variables in `args` ?

Reply:
No I will define a new env var for namespace and use that

Reply:
Done, please check again